src/flate/reader.cr

```
@@ -119,8 +125,12 @@ class Flate::Reader < IO
     end
   end
 
+  def unbuffered_flush
+    raise IO::Error.new "Can't flush Flate::Reader"
```

Shouldn't this be: `io.flush`? It makes no sense to raise here.

---

Well, it's a reader, but yeah, we can delegate

---

Do other readers raise on flush? I thought the default semantic for flush was to silently ignore if it has nothing to do.

---

Yes, other reads raise on flush.